e2e-test/resources/teaspoons/teaspoons_gcp_set_up_control_workspace.py

```diff
@@ -58,7 +58,6 @@
 auth_domains = [args.auth_domain]
 
 rawls_url = f"https://rawls.{env_string}"
-teaspoons_url = f"https://teaspoons.{env_string}"
```

Comment:
this is cleanup unrelated to this PR; teaspoons isn't called as part of this script so this variable isn't needed (and its presence is confusing)